api/Makefile

```diff
@@ -27,7 +27,6 @@ install:
     # Copy files and create folders
 	# ETC
 	$(INSTALL_DIR) $(ETC_INSTALLDIR)/api
-	$(INSTALL_RW_DIR) $(ETC_INSTALLDIR)/api/configuration
```

Comment:
Restore this line

Reply:
Fixed in cf533d856c1db1e32d99244bb707857b05e4f521.